I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/custom/login.rpy", line 132, in script
    if len(args[0]) < 5:
  File "game/custom/login.rpy", line 138, in <module>
    elif len(args[0] > 15):
TypeError: object of type 'bool' has no len()

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/custom/login.rpy", line 132, in script
    if len(args[0]) < 5:
  File "/Applications/renpy-6.99.12.4-sdk/renpy/ast.py", line 1656, in execute
    if renpy.python.py_eval(condition):
  File "/Applications/renpy-6.99.12.4-sdk/renpy/python.py", line 1749, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "/Applications/renpy-6.99.12.4-sdk/renpy/python.py", line 1743, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/custom/login.rpy", line 138, in <module>
    elif len(args[0] > 15):
TypeError: object of type 'bool' has no len()

Darwin-16.7.0-x86_64-i386-64bit
Ren'Py 6.99.12.4.2187
> turing 1.0a
